DependencyInjection/Configuration.php

```diff
@@ -213,6 +213,34 @@ public function getConfigTreeBuilder()
             ->end()
         ->end();
 
+        $rootNode
+            ->children()
+                ->arrayNode('webp')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('generate')
+                            ->defaultFalse()
+                            ->validate()
+                                ->ifTrue(function ($v) {
+                                    return !$v || function_exists('imagewebp');
```

Comment:
Shouldn't this be `!function_exists('imagewebp')`?

I've got webp enabled, but I get this error.

Reply:
I think too, i've created pull request #1322

Reply:
This code breaks logic and will result in fatal error.

```php
->ifTrue(function ($v) {
    return $v && !function_exists('imagewebp');
})
```

You enable WebP generation even though your PHP version does not support WebP generation.

```php
// read this as generate option is valid if generate option is disabled or enabled and WebP support by PHP
->ifTrue(function ($v) {
    return !$v || function_exists('imagewebp');
})
```